Pipfile

```
@@ -23,6 +23,7 @@ networkx = "~= 2.4"
 beautifulsoup4 = "~= 4.8"
 pyyaml = "~= 5.2"
 toml = "~= 0.10"
+sentry-sdk = {extras = ["falcon"], version = "0.14.2"}
```

This looks incorrect, but haven't tested. Either way it's a good idea to pin it a bit more loosely.

```suggestion
sentry-sdk = {extras = ["falcon"], version = "~= 0.14.2"}
```